es6/ops/random.js

```diff
@@ -68,14 +62,14 @@ class RandomFloat extends PlanOutOpRandom {
 }
 
 class RandomInteger extends PlanOutOpRandom {
+  compatRandomIntegerCalculation(minVal, maxVal) {
```

Comment:
I thought about that but then classes like ```Sample``` had multiple compatible operations to perform 😢